tests/benchdnn/brgemm/ref_brgemm.cpp

```diff
@@ -47,7 +56,12 @@ void compute_ref_brgemm(const prb_t *prb, const args_t &args) {
     const int64_t K = prb->k;
 
     // Using workspace memory as a method to get brgemm attributes.
+#if defined(brg_x64)
     using brgemm_attr_t = dnnl::impl::cpu::x64::brgemm_attr_t;
+#else
```

Comment:
```suggestion
#elif defined(brg_aarch64)
```

Reply:
@Shreyas-fuj  this one seems missed.

Reply:
Hi @dzarukin , thanks for pointing it out. I have corrected this. :)